### PR TITLE
Update integrations.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added documentation workflow for apps and plugins.

### CODE REVIEW

File: operations/research-and-development/engineering/integrations.md [hunk 1]
```diff
@@ -89,3 +89,19 @@ Seeing an intermittent connection issue with Zoom on http://dkh-local.ngrok.io.
 - Staff members add the issues there are working on for the current sprint to https://github.com/orgs/mattermost/projects/1.
     - Putting an issue is always preferred over putting a PR on the board.
 
+## Apps documentation workflow
+
+The Apps Framework and Marketplace documentation lives [here](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/integrate/apps). When submitting documentation updates and new documentation for apps, there are two options available:
+
+1. Submit the content with your PR in the [plugin-apps](https://github.com/mattermost/mattermost-plugin-apps) repo. Add the ``Docs/Needed`` label to the PR and ping @justinegeffen. The content will then be added to the docs, and the ``Docs/Done`` label added to your PR.
+
+2. Alternatively, you're welcome (and encouraged) to submit the docs PR yourself, [here](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/integrate/apps). then add a ``Docs/Needed`` label and tag @justinegeffen.
```
Comment (cwarnermm) [inline operations/research-and-development/engineering/integrations.md]:
```suggestion
2. Alternatively, you're welcome (and encouraged) to submit the docs PR yourself, [here](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/integrate/apps), then add a ``Docs/Needed`` label and tag @justinegeffen.
```

File: operations/research-and-development/engineering/integrations.md [hunk 2]
```diff
@@ -89,3 +89,19 @@ Seeing an intermittent connection issue with Zoom on http://dkh-local.ngrok.io.
...
+
+For both options, please ensure that doc updates are either provided by you or requested ahead of merge so there's time to review the content. At least two days' notice is ideal. The goal is to ensure that docs and releases are always in sync. 
+
+## Plugin documentation workflow
+
+The plugin documentation will soon be moved to developers.mattermost.com and will follow the same process as above. Until then, if you have a documentation update for your plugin, please work with @justinegeffen to ensure changes are made timeously ahead of release.
```
Comment (cwarnermm) [inline operations/research-and-development/engineering/integrations.md]: Consider making developers.mattermost.com an active link or formatting it for easy scanning.

Comment (cwarnermm) [reply operations/research-and-development/engineering/integrations.md]: This is the first time I've seen "timeously" used in a sentence. After looking up its definition, I agree that it's absolutely the best term based on intent. :)

**Review by cwarnermm — DISMISSED** [review]
Great updates! Minor feedback provided inline.

File: operations/research-and-development/engineering/integrations.md [hunk 3]
```diff
@@ -89,3 +89,19 @@ Seeing an intermittent connection issue with Zoom on http://dkh-local.ngrok.io.
...
+
+1. Submit the content with your PR in the [plugin-apps](https://github.com/mattermost/mattermost-plugin-apps) repo. Add the ``Docs/Needed`` label to the PR and ping @justinegeffen. The content will then be added to the docs, and the ``Docs/Done`` label added to your PR.
+
+2. Alternatively, you're welcome (and encouraged) to submit the docs PR yourself, [here](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/integrate/apps), then add a ``Docs/Needed`` label and tag @justinegeffen.
+
+For both options, please ensure that doc updates are either provided by you or requested ahead of merge so there's time to review the content. At least two days' notice is ideal. The goal is to ensure that docs and releases are always in sync. 
+
+## Plugin documentation workflow
+
+The plugin documentation will soon be moved to developers.mattermost.com and will follow the same process as above. Until then, if you have a documentation update for your plugin, please work with @justinegeffen to ensure changes are made timeously ahead of release.
```
Comment (justinegeffen) [inline operations/research-and-development/engineering/integrations.md]:
```suggestion
The plugin documentation will soon be moved to developers.mattermost.com and will follow the same process as above. Until then, if you have a documentation update for your plugin, please work with @justinegeffen to ensure changes made in time and aligned with release.
```

**Review by hanzei — APPROVED** [review]
LGMT :+1:

**Review by cwarnermm — APPROVED** [review]

**Review by aaronrothschild — APPROVED** [review]

**Pull request merged**